docs_src/design/ucr/URIs-for-Files.md

```diff
@@ -0,0 +1,60 @@
+
+- [Unit of Measure](https://github.com/edgexfoundry/edgex-go/blob/v2.3.0/cmd/core-metadata/res/uom.toml) file used by Core Metadata
+
+    - Location of this file is specified in configuration [here](https://github.com/edgexfoundry/edgex-go/blob/v2.3.0/cmd/core-metadata/res/configuration.toml#L50) 
+
+- Service Configuration files
+    - Location of these files are currently defaulted to be `./res/configuration.toml`, but can be overridden  via -cf/--configFile command line flag.
+    - The Common Configuration ADR adds a new [common configuration file](https://docs.edgexfoundry.org/3.0/design/adr/0026-Common%20Configuration/#specifying-the-common-configuration-location) specified via the future -cc/--commonConfig command line flag.
+
+- [Token Configuration](https://github.com/edgexfoundry/edgex-go/blob/v2.3.0/cmd/security-file-token-provider/res/token-config.json) file for Security File Token Provider 
+
+    - This file specifies the list of services to generate Vault tokens for. 
+
+- Device Profiles, Device Definition and Provision Watchers
+
+    - These files can reside in a device services local file system and are pushed to Core Metadata the first time the service starts. Example [here](https://github.com/edgexfoundry/device-onvif-camera/tree/v2.3.0/cmd/res)
+
+    - These files are found by scanning the folders specified in configuration [here](https://github.com/edgexfoundry/device-sdk-go/blob/main/example/cmd/device-simple/res/configuration.toml#L94-L96)
+
+    !!! note 
+        These files are only pushed to Core Metadata the first time the device service is loaded. They are not currently re-pushed once they exist in Core Metadata even when the files have changed locally. Thus updating the files locally or in a shared location will not result in changing the contents of these files in Core Metadata. They still benefit from this capability during initial deployment and when new files are added.  
+
+
+Currently all files loaded by services are expected to be on the local file system, thus are duplicated many times when deploying at scale.
+
+### Target Users
+- Software Deployer
+- Software Integrator
+
+### Description
+This UCR proposes to enhance loading of files in EdgeX by allowing the location of the file to be optionally specified as an URI.
+
+### Existing solutions
+Loading shared files via a URI is not new in the software industry. Here is the Wiki page for [Uniform Resource Identifier](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier) 
+
+### Requirements
+- Single EdgeX service files shall optionally be loaded via a specified URI. 
+- Sets of EdgeX service files (i.e. device service files) shall optionally be loaded via a URI. Details on how are left to the ADR.
+- The URIs specified shall follow the [Uniform Resource Identifier](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier) spec when authentication not required or using **basic-auth** in plain text in the URI,  i.e. `username:password@`
```

Comment:
Relevant comment: https://github.com/edgexfoundry/edgex-docs/pull/909#discussion_r1046133243

Reply:
I don't think it is necessary to specify, but if it would add clarity, some wording around "Certificate validation for https:// URLs will be performed using the system's built-in trust anchors."   That way, it is clear that you can't use a self-signed cert or a custom cert unless it is installed in the host (container) trust store.

Reply:
I'd also specify that minimum TLS version for the server is 1.3.   (Even AWS supports TLS 1.3 now, I don't think it is a stretch at this point to require it.)

Reply:
@bnevis-i , isn't just specifying HTTPS go enough for the UCR.? IMO, that other information is more appropriate for the ADR.

Reply:
Yes, that's ok to put it in the ADR.